src/document.js

```diff
@@ -604,13 +612,17 @@ const documentsMain = {
 				return
 			}
 
-			const list = result.data.ocs.data.map((user) => {
+			return result.data.ocs.data
+		}).catch(() => {
+			return documentsMain.users
+		}).then((users) => {
+			const list = users.map((user) => {
```

Comment:
Could we do this right away if there is no user session available so that we skip the failing request?

Reply:
Yep nice suggestion, I changed the logic around